src/lib/ui/ClimateSettingForm/ClimateSettingScheduleDeviceSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import { isThermostatDevice } from 'seamapi'
+
+import { DeviceTable, type UseDevicesData } from 'lib/index.js'
```

Comment:
I need to find some vscode config or something to change the resolution algorithm.
```suggestion
import { DeviceTable, type UseDevicesData } from 'lib/seam/devices/use-devices.js'
```

Reply:
haha just read this